code/src/index.css

```diff
@@ -1,13 +1,124 @@
+@import url('https://fonts.googleapis.com/css2?family=Nunito:wght@400;700&family=Poppins:wght@300;400;600&display=swap');
+
+:root {
+  --pink: #ed86be;
+  --peach: #eda686;
+  --blue: #869aed;
+  --grey: #a9a9a9;
+  --white: #ffffff;
+  --black: #000000;
+}
+
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
+  font-family: 'Poppins', sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
 }
 
-code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
-    monospace;
+.list-wrapper {
+  display: grid;
+  flex-direction: column;
+  align-items: center;
+  width: 100%;
+  min-height: 40%;
+  padding: 10% 5%;
+  background-color: #fff;
+  position: absolute;
+  top: 40%;
+  border-radius: 30px;
+}
+.flex-item {
+  display: grid;
+  grid-template-columns: repeat(5, 1fr);
+  align-items: center;
+  border-bottom: 1px dotted var(--blue);
+}
+
+.task {
+  grid-column: span 3;
+  word-break: break-word;
+}
+
+.delete-button {
+  grid-column: span 1;
+  padding: 2% 3%;
+  color: var(--blue);
+  background-color: var(--white);
+  border: 1px solid var(--blue);
+  border-radius: 10px;
+  font-family: 'Poppins', sans-serif;
+  font-size: 12px;
+  font-weight: 700;
+}
+
+.checkbox-label {
+  display: block;
+  position: relative;
+  padding-left: 25px;
+  margin: 0px 5px;
+  user-select: none;
+}
+.checkbox-label input {
+  position: absolute;
+  opacity: 0;
+  cursor: pointer;
+  height: 0;
+  width: 0;
+}
+.custom-checkbox {
+  position: absolute;
+  top: -11px;
+  left: 0;
+  height: 25px;
+  width: 25px;
+  background-color: var(--peach);
+  border-radius: 50%;
+}
```

Comment:
Amazing job with the checkboxes! It's impressive that you managed to do it without using any external libraries or such.